wremnants/CardTool.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, cardName="card.txt"):
         self.nominalTemplate = ""
         self.spacing = 28
         self.systTypeSpacing = 16
-        self.procColumnsSpacing = 30
+        self.procColumnsSpacing = 50
```

Comment:
Is this really needed? If it is too large the consequence is that a standard datacard will have a lot of empty space making it impossible to parse (sometimes it is useful to be able to read it by eye).
If you had to do it to avoid overlapping process names, this should now be fixed from PR https://github.com/WMass/WRemnants/pull/236

Reply:
Yes this was exactly the reason and thus no longer needed, I can revert this change, thanks for pointing out